### DISCUSSION TITLE
集成spring通过在application.yml指定app.id,apollo.meta等提示`Cannot resolve configuration property 'apollo.meta'`

### DISCUSSION THREAD

**Discussion by BobDu** [apolloconfig/apollo#3499] [Q&A]
如题
使用IntelJ IDEA做为开发工具
apollo-clinet版本 1.2.0  1.7.0均有尝试

集成spring通过在application.yml指定app.id,apollo.meta等时 
IDEA会有警告提示`Cannot resolve configuration property 'apollo.meta'`

**Comment by nobodyiam** [comment]
参照文档[1.2.2 Apollo Meta Server](http://ctripcorp.github.io/apollo/#/zh/usage/java-sdk-user-guide?id=%e4%b8%89%e3%80%81%e5%ae%a2%e6%88%b7%e7%ab%af%e7%94%a8%e6%b3%95)详细检查一下配置方式吧

**Reply by BobDu** [reply]
感谢收到您的答复
按照文档的方式配置  功能运行也是没有任何问题的  只是IntelJ IDEA会有警告提示
如图
![image](https://user-images.githubusercontent.com/20370175/104991949-df02ef00-5a5a-11eb-9a7f-1f40f508231f.png)

**Reply by nobodyiam** [reply]
这个是 idea 的智能提示，和运行时无关，可以忽略

**Reply by BobDu** [reply]
好的  很感谢您的帮助
但是IntelJ IDEA作为事实上使用最广泛的Java开发IDE  如果能支持IDEA的自动补全就更好了 更容易像我这样的Java新手上手使用
很抱歉我能力有限无法提供PR来实现这一功能
